Add volumetric flow rate value and add in water flow rate #688

christian-rli opened the pull request:
Closes #688 

Adds `volumetric flow rate value` with the definition "A volumetric flow rate value is a quantity value that has a volumetric flow rate unit as unit." and extends `water flow rate` by `'has quantity value' some 'volumetric flow rate value'`. Deletes `'has unit' some 'volumetric flow rate unit'` from `water flow rate`. If I did everyting right, that is :) This is my first implementation, so please have a closer look.

l-emele reviewed (changes requested):
Please add a term tracker item with the issue and pull request numbers and update CHANGELOG.md.

sfluegel05 commented:
It would also be a nice addition to have an axiom for `volumetric flow rate value` that says it is a subclass of `has unit some volumetric flow rate unit`.

christian-rli commented:
I added a term tracker item, updated the changelog and added the axiom to `volumetric flow rate value` that says it is a subclass of `has unit some volumetric flow rate unit`.

l-emele commented:
I only saw now that you made your changes mainly in the file oeo.omn instead of oeo-physical.omn.
@sfluegel05: What is the best way to move the changes into the right file?

christian-rli commented:
> I only saw now that you made your changes mainly in the file oeo.omn instead of oeo-physical.omn.
> @sfluegel05: What is the best way to move the changes into the right file?

Oh, sorry about that. I was wondering how protégé deals with that. I thought since it's loading these ontologies from the original, it somehow knows what goes where, but it didn't come to mind again when committing (only) oeo.omn. To work on oeo physical specifically, I would need to just open oeo-physical? If there is no easy way to move this with git I can also recreate the whole procedure from scratch and remove the first changes. there weren't too many steps involved.

l-emele commented:
I usually first open oeo.omn. This imports and loads all other relevant files. Then I switch to the module I want to work on: 
![grafik](https://user-images.githubusercontent.com/36884905/108818320-79ee6c00-75b9-11eb-939a-56ea857e8791.png)

sfluegel05 commented:
I guess the easiest way to solve this is to open the files in a text editor and just cut and paste the added part from oeo to oeo-physical.

christian-rli commented:
Okay, I tried to operate all changes made to oeo.omn over to oeo-physical.omn which was a bit of a challenge and I hope everything was carried over. At least it still works in protege and I believe I caught all the changes from the diff.

l-emele commented:
The diff for the oeo-physical looks good now. However there are still some leftovers in the oeo.omn. I think these should be deleted manually. Do you agree, @sfluegel05 ?

sfluegel05 reviewed (approved):
This doesn't make much of a difference for the ontology, but deleting the unused declarations might help keeping the oeo.omn file tidy, so I removed them.

christian-rli commented:
Thanks for the clean-up and being patient with me!
